### ISSUE TITLE
Video always loops

### ISSUE THREAD

**Issue by vlidholt** [ardera/omxplayer_video_player#3]
There seem to be no way to disable the videos from looping, or getting a callback when the video ends.

**Comment by ardera** [comment]
Yeah that's true, though it shouldn't be too hard to implement. There's no way of getting a notification by omxplayer when the video has ended, and you also can't just disable omxplayer looping; since then omxplayer would terminate on end and the last frame wouldn't be visible anymore.

So basically one would have to implement a timer that fires just before the video ends, pause the video and send a video player event to flutter. The timer also needs to be disabled / armed on pause / play, and optionally synced with the reported remaining playtime when flutter queries the playback position.

I'm busy right now, but I'll implement it when I have time
